content/v2.0/reference/cli/influx/pkg/stack/init.md

```diff
@@ -0,0 +1,28 @@
+---
+title: influx pkg stack init
+description: The 'influx pkg stack init' command initializes and InfluxDB stack.
+menu:
+  v2_0_ref:
+    name: influx pkg stack init
+    parent: influx pkg stack
+weight: 201
+v2.0/tags: [templates]
+---
+
+The `influx pkg stack init` command initializes and InfluxDB stack.
+
+## Usage
+```
+influx pkg stack init [flags]
+```
+
+## Flags
+| Flag                        | Description                     | Input type      | {{< cli/mapped >}} |
+|:----                        |:-----------                     |:----------:     |:------------------ |
+| `-h`, `--help`              | Help for the `init` command     |                 |                    |
+| `--json`                    | Output data as JSON             |                 |                    |
```

Comment:
ahh shooot, glad I saw this, I forgot to normalize the json flag here on this command with all the others that are coming up. The apply and summary commands were also updated to use json. The flag looks like the following:

```
   --json                       Output data as json; defaults false; Maps to env var $INFLUX_OUTPUT_JSON
```

apply flags updated here:
https://github.com/influxdata/influxdb/blob/master/cmd/influx/pkg.go#L100

summary flags updted here:
https://github.com/influxdata/influxdb/blob/master/cmd/influx/pkg.go#L326

this func will be used in the future to normalize all json/hide headers type of flags:

https://github.com/influxdata/influxdb/blob/master/cmd/influx/main.go#L444